docs/data/charts/line-demo/line-demo.md

```diff
@@ -31,3 +31,15 @@ components: LineChart, LineElement, LineHighlightElement, LineHighlightPlot, Lin
 ## LineChartConnectNulls
 
 {{"demo": "LineChartConnectNulls.js"}}
+
+## Line with predicition
+
+To show that some part of the data have a different status, you can vary the rendering.
```

Comment:
Maybe mention styling instead of rendering?

```suggestion
To show that parts of the data have different meanings, you can render stylised lines for each of them.
```